packages/react-celo/src/constants.tsx

```diff
@@ -249,6 +252,16 @@ export const PROVIDERS: {
       }
     },
   },
+  [SupportedProviders.CoinbaseWallet]: {
+    name: SupportedProviders.CoinbaseWallet,
+    type: WalletTypes.CoinbaseWallet,
+    description: 'Scan a QR code to connect your wallet',
+    icon: CoinbaseWallet,
+    canConnect: () => true,
+    showInList: () => true,
+    listPriority: () => Priorities.Popular,
+    supportedPlatforms: [Platform.Mobile, Platform.Web, Platform.Desktop],
```

Comment:
Desktop in this case refers to apps that have a native os app that is downloaded (like CeloTerminal)

For CBW i think its just web and mobile

Reply:
addressed in 580350715d319a2a0ed6b9b4a1aa334d76ff95d9